beacon-chain/powchain/sendDeposits_test.go

```diff
@@ -27,12 +28,13 @@ func generateValidators(count int64) map[string]*prysmKeyStore.Key {
 		}
 		validatorKeys[hex.EncodeToString(validatorKey.PublicKey.Marshal())] = validatorKey
 	}
+	return validatorKeys
 }
 
 //should I explicitly return err here?
 func sendDeposits(testAcc *contracts.TestAccount, validatorKeys map[string]*prysmKeyStore.Key,
 	numberOfDeposits int64) {
-	depositAmountInGwei := uint64(contracts.Amount32Eth.Int64())
+	depositAmountInGwei := contracts.Amount32Eth.Uint64();
```

Comment:
@nisdas I followed your advice and applied `contracts.Amount32Eth.Uint64();` Can you help me to fix this error?
/beacon-chain/powchain/sendDeposits_test.go:37:46: depositcontract.Amount32Eth.Uint64 undefined (type func() *big.Int has no field or method Uint64)